src/TextSerializationWriter.cs

```diff
@@ -20,7 +20,13 @@ public class TextSerializationWriter : ISerializationWriter, IDisposable {
     /// </summary>
     public TextSerializationWriter()
     {
-        writer = new(_stream);
+        writer = new(
+            _stream,
+            // Default encoding
+            encoding: System.Text.Encoding.UTF8,
+            // Default buffer size
+            bufferSize: 1024,
```

Comment:
```suggestion
```

using the named parameter and the default values, can't you just pass leave open?

Reply:
Yes in .NET, but .NET Framework build (part of your CI pipeline) failed. I guess .NET FX doesn't support named parameters to skip optional params.

Reply:
Resolving this for now as the parameters are not optional in netstandard2.0. 
https://learn.microsoft.com/en-us/dotnet/api/system.io.streamwriter.-ctor?view=netstandard-2.0#system-io-streamwriter-ctor(system-io-stream-system-text-encoding-system-int32-system-boolean)